x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_routing/new_routing_stream_entry.tsx

```diff
@@ -29,14 +29,22 @@ export function NewRoutingStreamEntry() {
     }
   }, []);
 
+  const { setLocalStreamName, isStreamNameValid, partitionName, prefix, helpText, errorMessage } =
+    useChildStreamInput(currentRule.destination, false);
+
```

Comment:
nit: we can drop the `false` argument here for consistency with `useChildStreamInput` other calls.